docs/en/observability/tab-widgets/uptime-monitoring/run-test-inline-content.asciidoc

```diff
@@ -0,0 +1,55 @@
+// tag::heartbeat[]
+
+Copy and paste your test steps into `heartbeat.yml`.
+{heartbeat} spawns a separate Node.js process, schedules your tests, and runs them on a Chromium browser.
+
+[source,yml]
+----
+- type: browser
+  id: elastic-website <1>
+  name: Elastic website
+  schedule: "@every 1m"
+  source:
+    inline:
+      script: |- <2>
+      step("load homepage", async () => {
+        await page.goto('https://www.elastic.co');
+      });
+      step("hover over products menu", async () => {
+        await page.hover('css=[data-nav-item=products]');
```

Comment:
As a note from someone who's in a country where GDPR applies: this test may fail for folks in this region of the world because the privacy notice covers the product nav item. It may be worth doing something somewhere else in the page as an example.

Reply:
Does it disable all elements on the page until you accept? I'm having trouble choosing a better example without being able to see the problem. 🙂

Reply:
It doesn't, the problem is that it covers that particular nav item at the top of the page. If you select any items _below_ the navbar it should be fine :)